selftests/unit/test_job.py

```diff
@@ -130,6 +130,25 @@ def post_tests(self):
         self.assertEqual(myjob.unique_id[::-1],
                          open(os.path.join(myjob.logdir, "reversed_id")).read())
 
+    @unittest.skip("Issue described at https://trello.com/c/qgSTIK0Y")
```

Comment:
Not really, because this is about the `tmp_dir`, and using job actually creates a `log_dir`, AKA "job results dir" per interpreter.